internal/service/dns_config/api_forward_zone_resource_test.go

```diff
@@ -0,0 +1,553 @@
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckForwardZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "external_forwarders.0.address", "192.168.11.11"),
+					resource.TestCheckResourceAttr(resourceName, "external_forwarders.0.fqdn", "tf-infoblox.com."),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccForwardZoneResource_ForwardOnly(t *testing.T) {
+	var resourceName = "bloxone_dns_forward_zone.test_forward_only"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccForwardZoneForwardOnly("tf-acc-test.com.", "false"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "forward_only", "false"),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccForwardZoneForwardOnly("tf-acc-test.com.", "true"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "forward_only", "true"),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccForwardZoneResource_Hosts(t *testing.T) {
+	var resourceName = "bloxone_dns_forward_zone.test_hosts"
+	var v dns_config.ConfigForwardZone
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccForwardZoneHosts("tf-acc-test.com."),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckForwardZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "fqdn", "tf-acc-test.com."),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccForwardZoneHosts("tf-acc-test.com."),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckForwardZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "fqdn", "tf-acc-test.com."),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccForwardZoneResource_InternalForwarders(t *testing.T) {
+	var resourceName = "bloxone_dns_forward_zone.test_internal_forwarders"
+	var v dns_config.ConfigForwardZone
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccForwardZoneInternalForwarders("tf-acc-test.com."),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckForwardZoneExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "fqdn", "tf-acc-test.com."),
```

Comment:
same comment as above, shouldn't you be testing hosts here?

Reply:
Resolved